rllib/algorithms/qmix/qmix_policy.py

```diff
@@ -324,8 +327,41 @@ def compute_actions_from_input_dict(
         return tuple(actions.transpose([1, 0])), hiddens, {}
 
     @override(TorchPolicy)
-    def compute_actions(self, *args, **kwargs):
-        return self.compute_actions_from_input_dict(*args, **kwargs)
+    def _compute_actions_without_connectors(
+        self,
+        obs_batch: Union[List[TensorStructType], TensorStructType],
+        state_batches: Optional[List[TensorType]] = None,
+        prev_action_batch: Union[List[TensorStructType], TensorStructType] = None,
+        prev_reward_batch: Union[List[TensorStructType], TensorStructType] = None,
+        info_batch: Optional[Dict[str, list]] = None,
+        episodes: Optional[List["Episode"]] = None,
+        explore: Optional[bool] = None,
+        timestep: Optional[int] = None,
+        **kwargs,
+    ) -> Tuple[TensorStructType, List[TensorType], Dict[str, TensorType]]:
+        with torch.no_grad():
```

Comment:
This code is very similar to the TorchPolicyV2 logic, but has to be rewritten here to fit QMix's special `_compute_actions_without_connectors_from_input_dict` after branching off the compute action functions here.